src/js/actions/libraries.js

```diff
@@ -620,9 +621,15 @@ define(function (require, exports) {
             .bind(this)
             .then(function (path) {
                 var layerRef = layerEffectAdapter.referenceBy.current,
-                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path);
-
-                return descriptor.playObject(placeObj);
+                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path),
```

Comment:
According the Seth, layer / text style should not apply to locked layers, so I switched to `playSimpleLayerActions` with overwrite lock turned off. Also filtered out locked layers before apply the style, so that this will works smoothly on multi selection of locked and unlocked layers.